glossary.md

```diff
@@ -27,14 +31,49 @@ For a sell order, this is the lower limit of what the seller is willing to accep
 
 **Market price:** For buy orders, this means that the buyer will is willing to pay the current market sell price. For sell orders, this means that the seller is willing to sell at the current market buy price. 
 
-**MeterReading:** Registration of the actual power consumed/injected by an asset
+**MeterReading:** Registration of the actual power consumed/produced by an asset
 
 **Order:** Flexibility order placed on a market. The order can be either “Buy” in case the market participant wants to procure flexibility or “Sell” in case the market participant wants to offer flexibility. 
 
-**PayAsBid:** This is a market rule stating that the price at which trade between two offers will occur is the proposed price of the first offer placed on the platform. 
+**PayAsBid:** This is a market rule stating that the price at which trade between two offers will occur is the proposed price of the first order placed on the platform. 
 
 **PayAsCleared:** This is a market rule stating that the price at which a trade will occur is the one of the market equilibrium (determined by the buy and sell curves). 
 
 **Portfolio:** A portfolio represents one or more assets (aggregated) that can participate in a flexibility market, e.g. batteries, dispatchable generators, etc. 
 
-**Reservation Market:** Flexibility market in which participants offer the reservation of their flexibility (possibly weeks or years in advance depending on market rules) before actual dispatching. If their reservation offer is activated, they are forced to place an offer on the consecutive activation market (see above)
+**QuantityType:** What is traded on a market / in an order. Possible values are 
+*ActivePower*, *ReactivePower*, *Energy*, *Capacity*. 
+
+**Reservation Market:** Flexibility market in which participants offer the reservation of their flexibility (possibly weeks or years in advance depending on market rules) before actual dispatching.
+The actual dispatching can be done outside the market or in a consecutive activation market. 
+
+**Trade:** A trade either between two participants of a market, or between one participant 
+and the FMO. A trade is always the result of two or more orders being matched. 
```

Comment:
is it possible to have a trade between a market participant and the FMO ?

Reply:
yes, and in NODES that is the default. Each trade is using the FMO as the middle man

Reply:
ok then :)